jakarta.md

```diff
@@ -0,0 +1,101 @@
+## Usage
+Usage guides for running tests against Jakarta and older EdgeX releases.
+
+### Run tests using checkbox-edgexfoundry snap
+This is the recommended method as it runs all the tests in isolation.
+
+Install from the beta channel:
```

Comment:
why do we need to install it from beta channel?

Reply:
Because the revision on edge is designed to work for latest tests only. It won't run the jakarta tests on the certified UC16 hardware (see the latest test reports).